### ISSUE TITLE
Visual bell, please

### ISSUE THREAD

**Issue by luebking** [lxqt/qterminal#1356]
##### Relevant Problems
Nobody wants their terminal to make funny noises, but there should™ be a response to \a
This is also an a11y problem, because there are people who cannot hear the funny noises their computer makes.

##### The Solution You'd Like
The feature is currently stashed in TerminalDisplay and the mode unconditionally set when creating QTermWidget
There two ways to address this
1. expose the TerminalDisplay setting via the QTermWidget API
2. https://doc.qt.io/qt-6/qgraphicseffect.html - which doesn't require qtermwidget patches and allows all sorts of fancy stuff (animated inversions, flashes, slaps, blurring, fish-eye… whatever you can come up with) but is kinda monkey-patching the feature on top.

This is basically an inquiry for the preferable approach.

**Comment by tsujan** [comment]
> Nobody wants their terminal to make funny noises

I like the current sound  :P 

> and allows all sorts of fancy stuff

In general, LXQt avoids fancy stuff in its codes. As for animations, there's only an optional one in lxqt-panel, because some users might find it unnatural or even annoying if their auto-hiding panel was shown or hidden suddenly.

**Comment by luebking** [comment]
I'm perfectly fine w/ a 150ms inversion flash - I just brought that up to point out the pros and cons and options.
Do you have a preference on how to enable the visual bell?

**Comment by tsujan** [comment]
I just told what I knew about LXQt; otherwise I like an optional animation :)

> I'm perfectly fine w/ a 150ms inversion flash

If it doesn't clutter the code, make a PR please.

**Comment by luebking** [comment]
I meant about either exposing the API in qtermwidget and using that or ignoring the terminaldisplay feature and implement a home-brew effect leveraging QGraphicsEffect

Not so much the style of preferred flashiness ;)
